runtime/core/test/event_tracer_test.cpp

```diff
@@ -109,6 +109,7 @@ class DummyEventTracer : public EventTracer {
     (void)end_time;
     (void)metadata;
     (void)metadata_len;
+    return Result<bool>(true);
```

Comment:
You can keep it, but actually you can just return a true rather than explicitly creating a Result holding a true.